checkmanga_tests.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import checkmanga
+import mock
+
+class TestCheckMangaUtil(unittest.TestCase):
+	def SetUp():
+		testDB = checkmanga.CheckMangaUtil('mytest.db')
```

Comment:
indents. also, great job all tests pass

Reply:
yeah I need to rewrite/finish the tests

Reply:
haha fuck me im a fuckin sarcastic piece o shit